pkg/pdapi/pdapi.go

```diff
@@ -40,6 +40,10 @@ import (
 
 const (
 	DefaultTimeout = 5 * time.Second
+
+	StoreStateUp        = "Up"
+	StoreStateOffline   = "Offline"
+	StoreStateTombstone = "Tombstone"
```

Comment:
can we use `TiKVStateTombstone` in `pkg/apis/pingcap/v1alpha1/types.go`?

Reply:
sure, updated